src/app/navigation/utils.spec.ts

```diff
@@ -241,3 +244,47 @@ describe('matchSearch', () => {
     ).toEqual('url3%3Farchive=https%3A%2F%2Farchive-content03.cnx.org');
   });
 });
+
+describe('isScrollTarget', () => {
+  it('return true if passed object is vali scroll target', () => {
+    expect(isScrollTarget({ type: 'some-type', elementId: 'elem' })).toEqual(true);
+  });
+
+  it('return false if passed object does not have elementId or type or has wrong type', () => {
+    expect(isScrollTarget({ elementId: '' })).toEqual(false);
+    expect(isScrollTarget({ notType: 'some-type', elementId: 'elem' })).toEqual(false);
+    expect(isScrollTarget({ type: 1, elementId: 'elem' })).toEqual(false);
+    expect(isScrollTarget({ type: 'type', elementId: 1 })).toEqual(false);
```

Comment:
You could decouple those assertions into separate tests which would give you clearer tests descriptions